content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -581,6 +483,165 @@ The following configuration variables are for features that are available for us
 : Enables incrementally flushing large traces to the Datadog Agent, reducing the chance of rejection by the Agent. Use only when you have long-lived traces or traces with many spans. Valid values are `true` or `false`. Added in version 1.26.0, only compatible with the Datadog Agent 7.26.0+.<br>
 **Default**: `false`
 
+## Custom instrumentation
+
+Getting started with custom instrumentation depends on which installation option was used to install automatic instrumentation.
+
+{{< tabs >}}
+
+{{% tab "Windows" %}}
+
+<div class="alert alert-warning">
+  <strong>Note:</strong> If you are using both automatic and custom instrumentation, it is important to keep the package versions (for example, MSI and NuGet) in sync.
+</div>
+
+To use custom instrumentation in your .NET application:
+1. Add the `Datadog.Trace` [NuGet package][1] to your application.
+2. In your application code, access the global tracer through the `Datadog.Trace.Tracer.Instance` property to create new spans.
+
+
+[1]: https://www.nuget.org/packages/Datadog.Trace
+{{% /tab %}}
+
+{{% tab "Linux" %}}
+
+<div class="alert alert-warning">
+  <strong>Note:</strong> If you are using both automatic and custom instrumentation, it is important to keep the package versions (for example, MSI and NuGet) in sync.
+</div>
+
+To use custom instrumentation in your .NET application:
+1. Add the `Datadog.Trace` [NuGet package][1] to your application.
+2. In your application code, access the global tracer through the `Datadog.Trace.Tracer.Instance` property to create new spans.
+
+
+[1]: https://www.nuget.org/packages/Datadog.Trace
+{{% /tab %}}
+
+{{% tab "NuGet" %}}
+
+To use custom instrumentation in your .NET application:
+1. In your application code, access the global tracer through the `Datadog.Trace.Tracer.Instance` property to create new spans.
+
+{{% /tab %}}
+
+{{< /tabs >}}
+
+For additional details on custom instrumentation and custom tagging, see the [.NET Custom Instrumentation documentation][3].
+
+## Configuring process environment variables
+
+### Windows
+
+#### Windows services
```

Comment:
Minor point, but I was wondering if we should a little explainer that this is only "how to configure env vars", and that the previous section describes _what_ environment variables to set?

Reply:
I've updated the text, let me know what you think

Reply:
LGTM!